pkg/dns-server/resolver/ip_test.go

```diff
@@ -0,0 +1,58 @@
+package resolver_test
+
+import (
+	"math"
+
+	. "github.com/Kong/kuma/pkg/dns-server/resolver"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("DNS ip", func() {
+
+	It("IP allocate and free operations", func(done Done) {
```

Comment:
Since we decided to follow the rule, please add "should" here: `It("should allocate and free IP")` or something like this